package_manager/dpkg_parser.py

```diff
@@ -72,59 +75,46 @@ def main():
                               args.packages_gz_url, args.package_prefix)
         util.build_os_release_tar(args.distro, OS_RELEASE_FILE_NAME, OS_RELEASE_PATH, OS_RELEASE_TAR_FILE_NAME)
     else:
-        download_dpkg(args.package_files, args.packages, args.workspace_name)
+        download_dpkg(args.package_files, args.packages, args.workspace_name, args.versionsfile)
 
-def download_dpkg(package_files, packages, workspace_name):
+def download_dpkg(package_files, packages, workspace_name, versionsfile):
     """ Using an unzipped, json package file with full urls,
      downloads a .deb package
 
     Uses the 'Filename' key to download the .deb package
     """
-    pkg_vals_to_package_file_and_sha256 = {}
     package_to_rule_map = {}
     package_to_version_map = {}
     package_file_to_metadata = {}
-    for pkg_vals in set(packages.split(",")):
-        pkg_split = pkg_vals.split("=")
-        if len(pkg_split) != 2:
-            pkg_name = pkg_vals
-            pkg_version = ""
-        else:
-            pkg_name, pkg_version = pkg_split
+    for pkg_name in set(packages.split(",")):
```

Comment:
Can you confirm it is no longer the case that items in `packages` are a "pkg_name=version" pair? Why do think the old code was attempting to parse the item?

Reply:
I've removed all of them in the workspace file. 

The only reason I can imagine the code was in there, is to support pinning the versions. The comments above the pinned packages in the WORKSPACE file (see diff) seem to suggest the same. 

But this doesn't make any sense to me given that the most recent version should be resolved when using the snapshots. Pinning is already done through snapshot pinning. So pinning both the snapshot and the package version can only result in errors when the two mismatch.

The only argument to leave the code in, would be to support other project that import these rules through GIT?

Reply:
Ah, now I get it where these values come from (`dpkg_list`, that is). I agree we don't need this anymore.